homeassistant/components/delijn/sensor.py

```diff
@@ -0,0 +1,107 @@
+"""Support for De Lijn (Flemish public transport) information."""
+import asyncio
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.exceptions import PlatformNotReady
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_NEXT_PASSAGE = 'nextpassage'
+CONF_STOP_ID = 'stop_id'
+CONF_SUB_KEY = 'sub_key'
+CONF_MAX_PASSAGES = 'max_passages'
+
+DEFAULT_NAME = 'De Lijn'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_SUB_KEY): cv.string,
+    vol.Required(CONF_NEXT_PASSAGE): [{
+        vol.Required(CONF_STOP_ID): cv.string,
+        vol.Optional(CONF_MAX_PASSAGES, default=5): cv.positive_int}]
```

Comment:
From my understanding are those the lines which are monitored. Would be something like `CONF_MAX_LINES` make more sense?

Reply:
I don't really follow this suggestion if that's ok for you. For me stations are more related to trains, for trams and busses "stops" is the correct terminology (and it's also used like this on the official site of De Lijn: https://www.delijn.be/en/haltes/).
With Passage I mean the next passage of a bus at a stop, in this case the default is that it will retrieve the next 5 passages of busses or trams at a stop. Which is also the correct terminology (and also here it's the terminology used on the official site of De Lijn: https://www.delijn.be/en/haltes/halte/200552/Gent_Korenmarkt_perron_2).
Hope that explains the use of this terminology.

Reply:
Do we have an existing corresponding config option in the other transport integrations, where we call this something else? Please review them and check. We need to make sure that we don't dilute this concept into different names.

Reply:
Thanks for the feedback.

I quickly went over all the components of the type [transport](https://www.home-assistant.io/components/#transport). This is what I came up with.

[entur_public_transport](https://www.home-assistant.io/components/entur_public_transport/) uses number_of_departures as variable which perhaps is the closes to what I'm using it for. Although personally I don't like "departure" as terminology for a bus or tram stop as it's not really grammatically correct.

[Mvglive](https://www.home-assistant.io/components/mvglive/) uses number as variable for the same purpose.

[Rmvtransport](https://www.home-assistant.io/components/rmvtransport/) uses max_journeys as a variable, but it's quite different from what I'm using it for. It seens that this integration really works with a "from" and "to" to define journeys (I want to go from point A to point B and that's my journey). So in that concept the max_journeys variable makes sense.

So bottom-line, yes it seems that the "number_of_departures" from entur_public_transport and "number" from Mvglive integrations are used at the moment for similar purposes. If you'd really like me to make the adjustment, I would personally vote for  "number_of_departures" as just "number" does not say much. 

Any advise/proposition?

Thanks in advance!

Reply:
`number_of_departures` sounds good!

Reply:
Changes made in last commit